src/components/AddBooks/AddBook.js

```diff
@@ -80,71 +99,74 @@ const AddBooks = () => {
         <input
           type="text"
           value={title}
-          onChange={handleTitleChange}
+          onChange = {(e) => setTitle(e.target.value)}
           placeholder="Enter title"
         />
         <input
           type="text"
           value={author}
-          onChange={handleAuthorChange}
+          onChange = {(e) => setAuthor(e.target.value)}
           placeholder="Enter author"
         />
         <input
           type="number"
           value={quantity}
-          onChange={handleQuantityChange}
+          onChange = {(e) => setQuantity(e.target.value)}
           placeholder="Enter quantity"
         />
```

Comment:
why dont we add radio buttons here to choose what collection we want to add the book to